osquery/utils/windows/lzxpress.cpp

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/logger/logger.h>
+#include <osquery/utils/expected/expected.h>
+#include <osquery/utils/windows/lzxpress.h>
+
+namespace osquery {
+namespace {
+
+#ifndef STATUS_SUCCESS
+#define STATUS_SUCCESS 0
+#endif
+
+typedef HRESULT(WINAPI* RTLDECOMPRESSBUFFEREX)(_In_ USHORT format,
```

Comment:
These two need to be NTSTATUS instead of HRESULT, to avoid confusion when someone looking at the return value in the future.

[1] https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-rtldecompressbufferex
[2] https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-rtlgetcompressionworkspacesize